drupal/sync/config_ignore.settings.yml

```diff
@@ -1,10 +1,10 @@
 ignored_config_entities:
-  0: 'webform.webform.*'
-  2: 'devel.*'
-  4: system.site
-  6: uhsg_service_provider_details.settings
-  8: 'xmlsitemap_engines.*'
-  10: 'google_tag.*'
+  - 'webform.webform.*'
+  - 'devel.*'
+  - system.site
+  - uhsg_service_provider_details.settings
+  - 'xmlsitemap_engines.*'
+  - 'google_tag.*'
```

Comment:
I might be using here new D8.8 `$settings['config_exclude_modules']` feature instead, CR: https://www.drupal.org/node/3079028

Reply:
So in `web/sites/default/settings.php` I should add following:
`$settings['config_exclude_modules'] = ['webform.webform', 'devel', 'xmlsitemap_engines', 'google_tag'];`

And from `drupal/sync/config_ignore.settings.yml` remove following:
  - 'webform.webform.*'
  - 'devel.*'
  - 'xmlsitemap_engines.*'
  - 'google_tag.*'

Reply:
`devel` only, other components needs to be present in remote environments. But `stage_file_proxy` and `update` are good candidates as well - all modules needed in local env only.